client/src/components/AdornedButton.js

```diff
@@ -0,0 +1,26 @@
+//AdornedButton.js
+
+import React from "react";
+import Button from "@material-ui/core/Button";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import withStyles from "@material-ui/core/styles/withStyles";
+
+const styles = {
+    root: {
+        marginLeft: 5
+    }
+};
+
+const SpinnerAdornment = withStyles(styles)(props => (
+    <CircularProgress className={props.classes.spinner} size={18} />
```

Comment:
Where are styles for 'spinner' class defined? I don't think they are being applied.